_testenv.yml

```diff
@@ -11,4 +11,7 @@ Password + Another Factor:
   CLIENT_SECRET: REPLACE_MFA_CLIENT_SECRET
 Custom Profile Enrolment Policy:
   CLIENT_ID: REPLACE_CUSTOM_CLIENT_ID
-  CLIENT_SECRET: REPLACE_CUSTOM_CLIENT_SECRET
\ No newline at end of file
+  CLIENT_SECRET: REPLACE_CUSTOM_CLIENT_SECRET
+Password and Google Authenticator Required:
+  CLIENT_ID: REPLACE_TOTP_CLIENT_ID
```

Comment:
Curious what's the TOTP clientId for?

Reply:
For scenarios 10.1.4-10.1.5 the special app should be used. This app have preconfigured profile enrollment policy "Google Authenticator Policy" which
- disables Email verification
- adds custom filed "Age" for sign-up
- auto assigns new users to the group "Google Authenticator Enrollment Required" (which is used for MFA policy with only required Google Authenticator)

Reply:
ic, so it's a clientId for okta app, I thought it's another util api, like a18n.